README.md

```diff
@@ -174,6 +161,12 @@ This is a complete list of all of the options that can be passed to `require("ob
     prepend_note_id = true
   },
 
+  -- Optional, keybindings.
+  keybindings = {
+    -- Overrides the 'gf' keybinding to work on markdown/wiki links within your vault.
+    gf_passthrough = true,
```

Comment:
This is exactly what I described in my issue suggesting this, but I'm now wondering if a more robust system might be preferable? For instance, like what can be found in [here](https://github.com/hrsh7th/nvim-cmp). 

Could potentially come up with a system like that for mappings to many of our commands with options (like passthrough) to modify their behavior from default. 

One the one hand, it seems a bit more transparent in case a user might want different keybind(s) than we set in the plugin, but it also might be a waste of effort if this is the only bind we ever set up.

Reply:
Actually I like that a lot. 53ea312